### PR TITLE
Increase the margin of the Our Projects section in the Engineering Team Page

### LINKED ISSUE #1372: Increase the margin of the Our Projects section in the Engineering Team Page
**Describe the bug**
Currently, Our projects section and the header part of the Engineering Team page is very close to each other, its like sticked them both.

**To Reproduce**
1. Go to 'Engineering Team Page'

**Expected behavior**
Add a margin top for the Our Projects section

**Screenshots**
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1372 

## Goals

Increasing the margin of the Our Projects section in the Engineering Team Page

## Approach

Increased the margin of the Our Projects section in the Engineering Team Page

### Screenshots

image
image

  
### Preview Link

https://pr-1380-sef-site.surge.sh/engineering-team.html

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by jayasanka-sack** [comment]
@RandilaP

File: engineering-team.html [hunk 1]
```diff
@@ -57,9 +57,9 @@ <h1 class="text-roboto text-capitalize pb-2">SEF Engineering Team</h1>
     </div>
 </section>
 <section class="container">
-    <div class="container my-5">
+    <div class="container my-5 py-5">
```
Comment (jayasanka-sack) [inline engineering-team.html]: Let's remove this one
```suggestion
    <div class="container my-5">
```

**Comment by jayasanka-sack** [comment]
I think we don't need this change. The current spacing looks okay to me. Shall we close this PR @YoshithaRathnayake ?

**Comment by YoshithaRathnayake** [comment]
> I think we don't need this change. The current spacing looks okay to me. Shall we close this PR @YoshithaRathnayake ?

Ok then

**Comment by jayasanka-sack** [comment]
Thanks @YoshithaRathnayake ! ❤️

**Comment by YoshithaRathnayake** [comment]
image

@jayasanka-sack is it ok if the Our Projects section visible in the bottom of the header?

**Comment by jayasanka-sack** [comment]
Yeah, that doesn't look good. However, increasing the margin is not the right solution to address this issue. 

ex: 
This is what it looks like on my laptop: 
Screenshot 2022-10-31 at 20 09 48

This is on my external monitor:

Screenshot 2022-10-31 at 20 10 56

We might need to consider `vh`. 

https://www.sitepoint.com/css-viewport-units-quick-start/
https://css-tricks.com/fun-viewport-units/

**Comment by YoshithaRathnayake** [comment]
> Yeah, that doesn't look good. However, increasing the margin is not the right solution to address this issue.
> 
> ex: This is what it looks like on my laptop: Screenshot 2022-10-31 at 20 09 48
> 
> This is on my external monitor:
> 
> Screenshot 2022-10-31 at 20 10 56
> 
> We might need to consider `vh`.
> 
> https://www.sitepoint.com/css-viewport-units-quick-start/ https://css-tricks.com/fun-viewport-units/

Oh then it's ok right? @jayasanka-sack

**Pull request closed without merging**